packages/toast/stories/toast.stories.ts

```diff
@@ -104,3 +108,74 @@ export const Negative = (args: Properties): TemplateResult =>
 
 export const Info = (args: Properties): TemplateResult =>
     variantDemo({ ...args, variant: 'info' });
+
+const overlayStyles = html`
+    <style>
+        html,
+        body,
+        #root,
+        #root-inner,
+        sp-story-decorator {
+            height: 100%;
+            margin: 0;
+        }
+
+        sp-story-decorator > div {
+            display: contents;
+        }
+
+        sp-story-decorator::part(container) {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            height: 100%;
+            align-items: center;
+            justify-content: center;
+        }
+
+        overlay-trigger {
+            flex: none;
+            margin: 24px 0;
+        }
+
+        .self-managed:nth-child(3) {
+            margin-left: 50px;
+        }
+    </style>
+`;
+
+const overlaid = (openPlacement: Placement): TemplateResult => {
+    return html`
+        ${overlayStyles}
+        ${(
+            [
+                ['bottom', ''],
+                ['left', 'negative'],
+                ['right', 'positive'],
+                ['top', 'info'],
+            ] as [Placement, string][]
+        ).map(([placement, variant]) => {
+            return html`
+                <overlay-trigger
```

Comment:
We probably should not insinuate that Toast elements can/should be opened on `hover`. I think if this was set up at `type="click"` and the slot was `slot="click-content"` that we'd be pretty good to go here!

Reply:
haha yes good point.

Reply:
addressed in [592bdbd](https://github.com/adobe/spectrum-web-components/pull/3511/commits/592bdbdf7b59f9b27fade2d47a56e1f63dba3f6f)